pep-0639.rst

```diff
@@ -1238,138 +1250,126 @@ specified under the same top-level key that require very different handling,
 and furthermore, unlike the previous arrangement, the subkeys were not mutually
 exclusive and can both be specified at once, and with some subkeys potentially
 being dynamic and others static, and mapping to different core metadata fields.
-This also breaks from the consensus for the core metadata fields, namely to
-separate the license expression into its own explicit field.
 
 Furthermore, this leads to a conflict with marking the key as ``dynamic``
-(assuming that is intended to specify :pep:`621` keys, as that PEP seems to rather
-imprecisely imply, rather than core metadata fields), as either both would have
-to be treated as ``dynamic``. A user may want to specify the ``expression``
-key as ``dynamic``, if they intend their tooling to generate it automatically;
-conversely, they may rely on their build tool to dynamically detect license
-files via means outside of that strictly specified here. And indeed, current
-users may mark the present ``license`` key as ``dynamic`` to automatically
-fill it in the metadata. Grouping all these uses under the same key forces an
-"all or nothing" approach, and creates ambiguity as to user intent.
+(assuming that is intended to specify the ``[project]`` table keys,
+as that PEP seems to imprecisely imply,
+rather than core metadata fields), as either both would have
```

Comment:
```suggestion
rather than core metadata fields), as either or both would have
```
Or maybe you mean something else? In any case the sentence is incomplete.